packages/core/src/app/payment/paymentMethod/PaymentMethodV2.tsx

```diff
@@ -28,6 +28,16 @@ export interface PaymentMethodProps {
     onUnhandledError(error: Error): void;
 }
 
+function shouldUsePaymentMethodV1(method: PaymentMethod, checkoutState: CheckoutSelectors) {
+    const useSquareOnPaymentMethodV1 =
```

Comment:
We should have this check on line 1 for Mollie, and return `true`. That way we don't need to execute the remaining code for square.